sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/BlobOutputStreamTest.groovy

```diff
@@ -139,6 +144,33 @@ class BlobOutputStreamTest extends APISpec {
         convertInputStreamToByteArray(blockBlobClient.openInputStream()) == data
     }
 
+    @Ignore
```

Comment:
Run live tests and check if this can be run live mode

Reply:
We need work item/issue to work on this infra.

Reply:
https://github.com/Azure/azure-sdk-for-java/issues/19601